spec/lib/query/proposal/search_spec.rb

```diff
@@ -83,7 +83,7 @@
       user = create(:user, client_slug: "test")
 
       proposal1 = create(:proposal, id: 199, requester: user)
-      test_client_request1 = create(:test_client_request, proposal: proposal1)
+      create(:test_client_request, proposal: proposal1)
```

Comment:
I think unused vars can be helpful in some cases but I don't think they are particularly enlightening in the particular context. let's keep as-is! :hamburger: